.github/workflows/ci.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       DB_DATABASE: wordpress_test
       DB_USER: root
       DB_PASSWORD: root
+      WP_VERSION: 6.0.3
```

Comment:
Could you specify this in the workflow's top level `env` block so all test are running with the same version of WP?